### ISSUE TITLE
Use requestAnimationFrame

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [FullScreenShenanigans/GamesRunnr#9]
![image](https://cloud.githubusercontent.com/assets/3335181/22002178/94553320-dbfe-11e6-8f3f-fa1f7d5eeec4.png)

FullScreenPokemon is capping out at about 200 FPS on a blank map or 180 FPS during battle. I suspect it's because using `setTimeout` won't allow timeouts of very small amount.

Instead, after #8, GamesRunnr should move that logic to a utility class that continuously calls `requestAnimationFrame` and triggers the games if enough time has passed.

**Comment by JoshuaKGoldberg** [comment]
Funny enough, `requestAnimationFrame` won't do it for tiny amounts. `setImmediate` would be better... if only modern browsers other than Edge implemented it. 😠 

```javascript
(() => {
    const now = performance.now();

    requestAnimationFrame(time => {
        console.log("rAF time", time - now);
    });

    setTimeout(() => {
        console.log("sTo time", performance.now() - now);
    });
})();
```
